### PR TITLE
TC-Wireframe & DB Integration & EC-Wireframe

### PR DESCRIPTION
**Pull request by paskantishubham**
This PR is in Ref to #50 && #54 && #55

### CODE REVIEW

**Comment by paskantishubham** [comment]
@ayasuda-ge, purpose of this PR has been satisfied. Please merge it. Thanks!

**Review by ayasuda-ge — APPROVED** [review]

File: webui-assets/src/Container/engagementRequest/EngagementRequest.js [hunk 1]
```diff
@@ -7,6 +7,9 @@ const TOKEN_URL =
 const FORM_SUBMIT_BASE_URL = "https://forms.smartsheet.com/";
 const FORM_SUBMIT_PATH = "api/submit/74f7e5c706d04aca841df70661a5631b";
 
+const BASE_URL ;
+const TOKEN = "d6d83337-e656-4fe5-96e0-ed449c749b72"
```
Comment (ayasuda-ge) [inline webui-assets/src/Container/engagementRequest/EngagementRequest.js:11]: This must go away.

**Comment by paskantishubham** [comment]
@ayasuda-ge I commented the unused lines(for future ref, it will be deleted later), please review latest commits.

**Comment by ayasuda-ge** [comment]
> @ayasuda-ge I commented the unused lines(for future ref, it will be deleted later), please review latest commits.

it is actually more complex than you may think. This commits cannot be part of the repo due to a failing security policy checker. Please consider rebase, and remove the credential completely from your commits. @paskantishubham

**Pull request closed without merging**